.github/workflows/pr.yml

```diff
@@ -10,7 +10,7 @@ on:
   pull_request:
     paths-ignore:
       - 'docs/**'
-    branches: [ main, v1, v2, next ]
+    branches: [ main, main-c5, v1, v2, next ]
```

Comment:
I think a more explicit name would be better, like `main-f8-5.12` or something similar?

Reply:
ok, make sense, maybe not put the minor number of the fabric8 client version, since it might change, something like

`main-f8-5.x` ?

Reply:
Done.